tools/multigps/multigps.xml

```
@@ -0,0 +1,457 @@
+                    --exclude '$aoc.exclude'
+                #end if
+                ## MultiGPS priors
+                #if str($umc.use_motif) == 'yes':
+                    #set mpc = $umc.multigps_priors_cond
+                    #if str($mpc.multigps_priors) == 'yes':
+                        #set bmc = $mpc.both_motifs_cond
+                        #if str($mpc.noposprior) == 'no':
+                            --noposprior
+                        #end if
+                        --probshared $mpc.probshared
+                        #if str($bmc.nomotifs) == 'yes':
+                            --memenmotifs $bmc.memenmotifs
+                            --mememinw $bmc.mememinw
+                            --mememaxw $bmc.mememaxw
+                        #else:
+                            #set mfoc = $bmc.nomotifprior_cond
+                            --nomotifs
+                            --nomotifprior $mfoc.nomotifprior
+                            #if str($mfoc.nomotifprior) == 'yes':
+                                --memenmotifs $mfoc.memenmotifs
+                                --mememinw $mfoc.mememinw
+                                --mememaxw $mfoc.mememaxw
+                            #end if
+                        #end if
+                    #end if
+                #end if
+                ## Reporting binding events
+                #if str($rbec.report_binding_events) == 'yes':
+                    --q $rbec.minqvalue
+                    --minfold $rbec.minfold
+                    #if str($rbec.nodifftests) == 'no':
+                        --nodifftests
+                    #end if
+                    --edgerod $rbec.edgerod
+                    --diffp $rbec.diffp
+                #end if
+            #end if
+            2>&1
+            --out '$output_html.files_path'
+            && cp $output_dir/*.events.txt '$experiment_events' || true
+            && mv $output_dir/*.html '$output_html' || true
```

`|| true` suppresses all possible errors that `cp` can give (full disk, wrong permissions) while you want those errors to be captured and reported